app/models/Organisation.js

```diff
@@ -81,20 +85,46 @@ const orgSchema = new Schema({
     ]
   },
   adminInfo: {
-    type: Object,
-    required: true,
+    name: {
+      type: String,
+      required: true
+    },
+    githubHandle: {
+      type: String,
+      required: true
+    },
+    slackHandle: {
+      type: String
+    },
+    others: [{}],
     validate (adminInfo) {
       if (validator.isEmpty(adminInfo)) {
         throw new Error('Admin info is required!')
       }
+      if (validator.isEmpty(adminInfo.githubHandle)) {
+        throw new Error('Github Handle of admin is required!')
+      }
     }
   },
   moderatorInfo: {
-    type: Object,
-    required: true,
-    validate (adminInfo) {
-      if (validator.isEmpty(adminInfo)) {
-        throw new Error('Admin info is required!')
+    name: {
```

Comment:
same with moderator, I think I have given in schema that its references a `Users` collection.

Reply:
It's not there, So I thought of this.
So should I make reference to the ```Users``` Collection instead of writing this?